pom.xml

```diff
@@ -1658,6 +1660,70 @@
                 <artifactId>poi-ooxml</artifactId>
                 <version>4.1.1</version>
             </dependency>
+
+            <!-- Amazon AWS SDK Versioning -->
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
```

Comment:
Is this v1 of the SDK or v2 ? AWS suggests we use the newer v2. https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html 

Can we switch ?